api/tests/devcomp/unit/infrastructure/factories/element-for-verification-factory_test.js

```diff
@@ -64,26 +64,56 @@ describe('Unit | Devcomp | Infrastructure | Factories | ElementForVerification',
       });
     });
 
-    it('should instantiate a EmbedForAnswerVerification when given a data of type "embed"', function () {
-      // given
-      const feedbacks = { valid: 'valid', invalid: 'invalid' };
+    describe('When element is an embed', function () {
+      describe('When embed has required completion', function () {
+        it('should instantiate a EmbedForAnswerVerification when given a data of type "embed"', function () {
```

Comment:
Redondant avec le `describe` :

```suggestion
        it('should instantiate a EmbedForAnswerVerification', function () {
```